### ISSUE TITLE
Question about the checkpoint

### ISSUE THREAD

**Issue by Jazz1996** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#61]
Thanks for your wonderful project!
I tried to train the model using your way. Firstly I trained only the decoder with the batch size of 80 without fine-tune for 15 epochs. Then I tried to use fine-tune and recover to the checkpoint, but after I added the checkpoint path and ran it again, the training didn't begin. Can you help me solve this problem? Thanks!
<img width="570" alt="屏幕快照 2019-04-03 下午2 30 32" src="https://user-images.githubusercontent.com/26078701/55457648-16ea3980-561d-11e9-8dc5-f8d3d15d7074.png">

**Comment by sgrvinod** [comment]
Hi @Jazz1996 , does this still happen? If it does, let's try to figure out why and if there's a bug, I'd like to fix it. 

When this happened, `checkpoint` was set to the path of the un-fine-tuned checkpoint and `fine_tune_encoder` was set to `True`, correct?

**Comment by Jazz1996** [comment]
Hi @sgrvinod , correct, and the training still doesn't begin.

**Comment by feixiangqiqi** [comment]
Hi @Jazz1996 ,Have you solved this problem, I hope I can get your help

**Comment by aliciaviernes** [comment]
Hello, is there any solution to this problem?

**Comment by Jazz1996** [comment]
It was a long time, so I can hardly remember this problem now. Sorry I do not have a solution to it.

**Comment by supriamir** [comment]
is there any solution to this problem?

**Comment by LavanyaGovindaraju** [comment]
Yes, check your training loop. Maybe you'll find the range something like (10,10). So that's why it doesn't show any error. Check what value is assigned to start_epoch.

**Comment by Qiqi224** [comment]
Hello，the epoch you set in your new train should bigger than the one in your last train.
